example.js

```diff
@@ -0,0 +1,28 @@
+var selectorPlatform = 

```

Comment:
Not used at all. This is just to show you what the function would look like before it is turned into a string if I were to use the function to string prototype method. I was curious as to how you were thinking this could be done since there are variables like Platform on line 7 that need to be changeable and entire lines that need to be add/removable.